### DISCUSSION TITLE
openroad gui

### DISCUSSION THREAD

**Discussion by msingh9** [The-OpenROAD-Project/OpenROAD#3068] [Q&A] [answered]
If ran openroad without gui, can I bring gui on later on after few steps from flow.tcl? I could save the design db and then open that design in gui, but I am just wondering if I can bring gui on from within openroad prompt.

1) openroad -log log
2) source <my_design>.tcl
3) After few steps of flow
4) I want to open the design in gui

**Comment by QuantamHD** [comment, marked as answer]
You can open the gui at anytime with `gui::show`

**Comment by msingh9** [comment]
This works like a charm. Thank you for comment.

**Comment by maliberty** [comment]
FYI you can go the other way with "Hide GUI":
![image](https://user-images.githubusercontent.com/761514/227319100-b11dc3c6-bf82-4464-aac2-318293c43c13.png)
